sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -530,8 +536,9 @@
 				D1CD93E7258D6F0600409613 /* updateMessage.json */,
 				D1CD93EB258D6F1300409613 /* updateTopic.json */,
 			);
-			path = Recordings;
-			sourceTree = "<group>";
+			name = Recordings;
+			path = "/Users/jairmyree/Desktop/azure-sdk-for-ios/sdk/communication/AzureCommunicationChat/Tests/Util/Recordings";
```

Comment:
Is there a way to make this path relative? I suspect it won't work for me when I pull this change.

Reply:
I believe that's possible, but this was not meant to be commited

Reply:
You can revert this file and I can see what happens on my end